packages/theming/default-theme/src/defaultColors.ts

```diff
@@ -263,14 +273,21 @@ export function getStockWebDarkPalette(): ThemeColorDefinition {
       {
         // colors taken from fluentui DarkCustomizations.ts
         themeDarker: '#82c7ff',
-        themeDark: globalTokens.color.brand.tint30,
-        themeDarkAlt: globalTokens.color.brand.tint20,
-        themePrimary: globalTokens.color.brand.tint10,
-        themeSecondary: globalTokens.color.brand.primary,
+        // @ts-ignore
+        themeDark: globalTokens.color.brand40,
+        // @ts-ignore
+        themeDarkAlt: globalTokens.color.brand40,
+        // @ts-ignore
+        themePrimary: globalTokens.color.brand40,
+        // @ts-ignore
+        themeSecondary: globalTokens.color.brand40,
         themeTertiary: '#235a85',
-        themeLight: globalTokens.color.brand.shade30,
-        themeLighter: globalTokens.color.brand.shade50,
-        themeLighterAlt: globalTokens.color.brand.shade60,
+        // @ts-ignore
+        themeLight: globalTokens.color.brand40,
+        // @ts-ignore
+        themeLighter: globalTokens.color.brand40,
+        // @ts-ignore
+        themeLighterAlt: globalTokens.color.brand40,
```

Comment:
Why do we need all these @ts-ignore comments?

Reply:
I would also rather we group all these together to one chunk so its more obvious "these tokens are non-standard"

Reply:
Because ESLint is referring the design-tokens-windows global tokens file, which has a different schema for color brand tokens (i.e. globalTokens.color.brand.shade30 is defined in win32/macOS, but globalTokens.color.brand30 is defined in mobile)

Also - thanks for making this comment because I just realized I put brand40 in a bunch of places as a placeholder since I didn't know what the equivalent was (ex. for globalTokens.color.brand.primary) - this should be fixed before I merge. I'll start a conversation about this offline

Reply:
Also just to add on - if we're okay with some global tokens having different schema across platforms, I don't know if there's a way around 1) the suggested/autocomplete values for the token values is sometimes wrong, and 2) having to add $ts-ignore comments, because ESLint is always assuming we're using the windows design-tokens package

Reply:
Ex. if you go to any reference to globalTokens.<something> and click go to reference, it always takes you to the windows design-tokens package in node modules

Reply:
Sure I can start making that change in the tokens repo, and once that change makes it into a new package I can do another update in FURN. I think we'll still need @ts-ignore comments, but they would be in the .win32.ts/.macOS.ts file (although it sounds like we want to update macOS to match web as well)

Reply:
Just thinking around using desing-tokens-windows acting as the type files for all endpoints - can we instead creating a `globalToken` type file in FURN - that would be a superset of types instead of  desing-tokens-windows acting as it? Unless we really use the design-tokens-windows for more reasons. 
Not a review on this PR  - but something for a thought ahead.

Reply:
Made an ADO item to update the windows schema file

Reply:
Saad do you think still we should "group all these together to one chunk so its more obvious "these tokens are non-standard"? Otherwise will resolve this thread

Reply:
Should be grouped together now